src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -489,7 +489,9 @@ function MoneyRequestConfirmationList({
 
     // Calculate and set tax amount in transaction draft
     const taxableAmount = isDistanceRequest ? DistanceRequestUtils.getTaxableAmount(policy, customUnitRateID, distance) : (transaction?.amount ?? 0);
-    const taxPercentage = getTaxValue(policy, transaction, transaction?.taxCode ?? defaultTaxCode) ?? '';
+    const taxPercentage =
+        getTaxValue(policy, transaction, transaction?.taxCode ?? defaultTaxCode) ??
+        (isMovingTransactionFromTrackExpense ? getTaxValue(policyForMovingExpenses, transaction, transaction?.taxCode ?? defaultTaxCode) : '');
```

Comment:
```suggestion
    const taxPercentage =
        getTaxValue(isMovingTransactionFromTrackExpense ? policyForMovingExpenses: policy, transaction, transaction?.taxCode ?? defaultTaxCode)
```

Reply:
@parasharrajat no, the condition is ok - first we want to check if we get the taxValue for the chosen policy, otherwise we would always get the taxValue for policyForMovingExpenses. If there's no tax value we fallback to this another condition

Reply:
I've added a comment to make it clear